ultraplot/constructor.py

```diff
@@ -845,65 +850,103 @@ def Cycle(*args, N=None, samples=None, name=None, **kwargs):
     ultraplot.constructor.Norm
     ultraplot.utils.get_colors
     """
-    # Parse keyword arguments that rotate through other properties
-    # besides color cycles.
-    props = _pop_props(kwargs, "line")
-    if "sizes" in kwargs:  # special case, gets translated back by scatter()
-        props.setdefault("markersize", kwargs.pop("sizes"))
-    samples = _not_none(samples=samples, N=N)  # trigger Colormap default
-    for key, value in tuple(props.items()):  # permit in-place modification
-        if value is None:
-            return
-        elif not np.iterable(value) or isinstance(value, str):
-            value = (value,)
-        props[key] = list(value)  # ensure mutable list
 
-    # If args is non-empty, means we want color cycle; otherwise is black
-    if not args:
-        props.setdefault("color", ["black"])
+    def __init__(self, *args, N=None, samples=None, name=None, **kwargs):
+        self.name = "_no_name"  # default value
```

Comment:
This is a change in behavior. In the previous code, the cycler always got the input name unless the function hit the early return. I'd use the `_not_none` here too.

Reply:
Right again!